fvm/environment/meter.go

```diff
@@ -48,12 +48,14 @@ const (
 	ComputationKindGetOrLoadProgram           = 2034
 	ComputationKindGenerateAccountLocalID     = 2035
 	ComputationKindGetRandomSourceHistory     = 2036
+	ComputationKindEVMGasUsage                = 2037
 )
 
 type Meter interface {
 	MeterComputation(common.ComputationKind, uint) error
 	ComputationUsed() (uint64, error)
 	ComputationIntensities() meter.MeteredComputationIntensities
+	ComputationAvailable(common.ComputationKind, uint) bool
```

Comment:
Should the uint size be explicitly defined so there's no weird things going on on different arch?
```suggestion
	ComputationAvailable(common.ComputationKind, uint64) bool
```

Reply:
I kept it similar to `MeterComputation(common.ComputationKind, uint)`, I don't think inputs to the methods would changed the behaviour based on the types. @janezpodhostnik thoughts? Even if we decide to change I would change them all together in another PR.